doc/api.yml

```diff
@@ -2451,6 +2451,40 @@ paths:
           $ref: '#/components/responses/429'
         '500':
           $ref: '#/components/responses/500'
+  '/school/files/share_link':
+    post:
```

Comment:
i assume that "POST" method was chosen because every call to this api will create a new link ?

Reply:
yes